src/showcase/src/pages/components/index.astro

```diff
@@ -0,0 +1,25 @@
+---
+const allComponents = await Astro.glob("./*.astro");
+---
+
+<html lang="en">
+  <head>
+    <title>II - Components</title>
+  </head>
+  <ul>
+    {
+      allComponents.map((C) => (
+        <li>
+          <div>
+            <C.default />
+          </div>
+          <a style="display: block; text-align: center;" href={C.url}>
```

Comment:
you can use a style tag to avoid repetition of the style in the page

Reply:
Yeah, but for now this is good enough I think :)